mmv1/products/workstations/terraform.yaml

```
@@ -76,34 +76,39 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         name: "workstation_config_basic"
         min_version: beta
         primary_resource_id: "default"
+        primary_resource_name: "fmt.Sprintf("tf-test-workstation-cluster%s", context["random_suffix"]), fmt.Sprintf("tf-test-workstation-cluster-config%s", context["random_suffix"])"
```

You need to escape all the double-quotes except the outermost pair.

```suggestion
        primary_resource_name: "fmt.Sprintf(\"tf-test-workstation-cluster%s\", context[\"random_suffix\"]), fmt.Sprintf(\"tf-test-workstation-cluster-config%s\", context[\"random_suffix\"])"
```

---

I was able to reproduce the error by [generating the providers locally](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/generate-providers/) - I'd recommend also making sure that you can run the unit tests just to make sure everything compiles: https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests